### ISSUE TITLE
Libary support for functional call [JAVA]

### ISSUE THREAD

**Issue by larycoder** [ComLake/core#2]
I don't want to call the raw HTTP request to your core, could you develop a lib so I can import to my project

**Comment by McSinyx** [comment]
> I don't want to call the raw HTTP request to your core,
> could you develop a lib so I can import to my project?

I don't see that happening in a foreseeable future, sorry.

**Comment by McSinyx** [comment]
Yes and no: it is a supported use case, but don't hold your breath on it :wink:
